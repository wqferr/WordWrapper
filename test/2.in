6
10
gi
fcfhgg
gcdid
lhe
l
b
gbh
ell
d
feiaa
