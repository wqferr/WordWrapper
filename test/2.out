gi
fcfhgg
gcdid
lhe l
b gbh
ell d
feiaa
